cpp/modmesh/view/R3DWidget.cpp

```diff
@@ -99,14 +99,13 @@ void R3DWidget::resizeEvent(QResizeEvent * event)
     m_container->resize(event->size());
 }
 
-void R3DWidget::resetCamera(Qt3DRender::QCamera * camera,
-                            float positionX,
-                            float positionY,
-                            float positionZ)
+void R3DWidget::resetCamera() const
 {
+    Qt3DRender::QCamera * camera = m_view->camera();
+
     // Set up the camera.
     camera->lens()->setPerspectiveProjection(45.0f, 16.0f / 9.0f, 0.1f, 1000.0f);
-    camera->setPosition(QVector3D(positionX, positionY, positionZ));
+    camera->setPosition(QVector3D(0.0f, 0.0f, 10.0f));
     camera->setViewCenter(QVector3D(0.0f, 0.0f, 0.0f));
     camera->setUpVector(QVector3D(0.f, 1.f, 0.f));
 
```

Comment:
refactored resetCamera function

Reply:
yes, we can do that. I think we need to expose more camera's props to python: setting camera's position, view center, etc. And we should move all camera's logic to CameraController class. Right now camera's logic is placed on several classes: e.g. camera reset is located in R3DWidget

Reply:
That should be a simple enhancement, I can do that in the next PR

Reply:
Thanks.  I added a bullet under issue #70 to track it.